crates/uv/src/commands/pip_list.rs

```diff
@@ -70,6 +79,205 @@ pub(crate) fn pip_list(
         return Ok(ExitStatus::Success);
     }
 
+    if outdated {
+        let constraints: &[RequirementsSource] = Default::default();
+        let overrides: &[RequirementsSource] = Default::default();
+        let extras: ExtrasSpecification<'_> = ExtrasSpecification::default();
+        let connectivity: Connectivity = Connectivity::Online;
+
+        let resolution_mode: ResolutionMode = ResolutionMode::Highest;
+        let prerelease_mode: PreReleaseMode = PreReleaseMode::default();
+        let dependency_mode: DependencyMode = DependencyMode::default();
+        let upgrade: Upgrade = Upgrade::All;
+        let index_locations: IndexLocations = IndexLocations::default();
+        let keyring_provider: KeyringProvider = KeyringProvider::default();
+        let reinstall: Reinstall = Reinstall::All;
+        let setup_py: SetupPyStrategy = SetupPyStrategy::default();
+        let config_settings: ConfigSettings = ConfigSettings::default();
+        let no_build: NoBuild = NoBuild::None;
+        let no_binary: NoBinary = NoBinary::None;
+        let exclude_newer: Option<DateTime<Utc>> = None;
+        let break_system_packages: bool = false;
+        let native_tls: bool = false;
```

Comment:
In order to run the package resolver, we set up quite a lot of defaults. In `pip_install`, these are all available as arguments on the command line. If we go with the approach of using the pip-install style resolve it implies we should probably add all of these as options for `pip list` 🤔 This strikes me as perhaps less than ideal.

I wonder if that's the path we want to go down?

Reply:
Yeah that seems less than ideal.

I wonder if `pip install --upgrade --report` should be used instead of `pip list --outdated` for your use-case... hm.